eng/Versions.props

```diff
@@ -62,15 +62,15 @@
   -->
   <PropertyGroup Label="Automated">
     <!-- Packages from dotnet/arcade -->
-    <MicrosoftDotNetGenAPIPackageVersion>1.0.0-beta.20124.2</MicrosoftDotNetGenAPIPackageVersion>
+    <MicrosoftDotNetGenAPIPackageVersion>1.0.0-beta.20113.5</MicrosoftDotNetGenAPIPackageVersion>
     <!-- Packages from dotnet/roslyn -->
-    <MicrosoftNetCompilersToolsetPackageVersion>3.4.1-beta4-19614-01</MicrosoftNetCompilersToolsetPackageVersion>
+    <MicrosoftNetCompilersToolsetPackageVersion>3.4.1-beta4-20127-10</MicrosoftNetCompilersToolsetPackageVersion>
```

Comment:
I'm not positive but it looks very strange to make _manual_ changes to the automated part of eng/Versions.props.

@wtgodbe_ @mmitche these are included in eng/Version.Details.xml as `Pinned="true"` dependencies. Since @pranavkm is updating them, should they be completely removed from that file?

Reply:
Yeah, if we're only going to update them manually, they should be removed from versions.details.xml

Reply:
Sure. I usually just snapshot these from the 3.1.3 branch. I could leave this as-is if you'd prefer that over manual updating it.